src/bluesearch/database/topic.py

```diff
@@ -0,0 +1,228 @@
+# Blue Brain Search is a text mining toolbox focused on scientific use cases.
+#
+# Copyright (C) 2020  Blue Brain Project, EPFL.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+"""Utils for journal/articles topics."""
+import pathlib
+from typing import Dict, Iterable, List, Union
+from xml.etree.ElementTree import Element  # nosec
+
+import requests
+from defusedxml import ElementTree
+
+
+# Journal Topic
+def get_mesh_from_nlm_ta(nlm_ta: str) -> List[Dict[str, Union[str, List[str]]]]:
+    """Retrieve Medical Subject Heading from Journal's NLM Title Abbreviation.
+
+    Parameters
+    ----------
+    nlm_ta : str
+        NLM Title Abbreviation of Journal.
+
+    Returns
+    -------
+    mesh :
+    """
+    nlm_ta_api = "+".join(nlm_ta.split(" "))
+    url = (
+        "https://www.ncbi.nlm.nih.gov/nlmcatalog/?"
+        f"term={nlm_ta_api}%5Bta%5D&report=xml&format=xml"
+    )
+
+    response = requests.get(url)
+    if not response.ok:
+        raise ValueError("The request did not work")
+
+    # The response is a fake xml,
+    # we need to change some characters of the response to have a valid xml.
+    text = (
+        response.content.decode()
+        .replace("<pre>", "")
+        .replace("</pre>", "")
+        .replace("\n", "")
+        .replace("&lt;", "<")
+        .replace("&gt;", ">")
+        .replace("><", ">    <")
+    )
+
+    print(text)
```

Comment:
Should it be removed?

Reply:
Done in 87525ff. Thanks for catching this.